libraries/cmake/facebook/modules/Findjinja2.cmake

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2014-present, Facebook, Inc.
```

Comment:
Built-time dependencies like this one are only required when generating table code from the spec files. I pretty much ignored them for now

Reply:
We can maybe track this into an issue and consider fixing this in a future release